### PR TITLE
WIP update hsd from v4.0.1 to v4.0.2

### PR DESCRIPTION
**Pull request by handshake-enthusiast**
just a reminder, see 667efb0b4a9cff924978849ba532a09c0ca8cd38

### CODE REVIEW

File: download/index.html [hunk 1]
```diff
@@ -126,7 +126,7 @@ <h1 class="low-margin">Download</h1>
 b2dd2438f8552c880a4659cacb41ce59108d07bce08c291c6a690abe4f7bdfdc  hs-airdrop-0.10.0.tar.gz<br>
 3852296b992e0e9ab61017fc46b59fc570017cefe0d5b5d23822fd4d1391feb7  hs-client-0.0.11.tar.gz<br>
 67ff3517f24194b4e1375c565634d07e263ddcbee1c744e82592b6cb8e31f780  hs-miner-0.0.10.tar.gz<br>
-7d5ca7be4068d4ee3c737ccc8e6c62a7bbeb7d69e0dbc100612e65e090190c55  hsd-4.0.1.tar.gz
+7d5ca7be4068d4ee3c737ccc8e6c62a7bbeb7d69e0dbc100612e65e090190c55  hsd-4.0.2.tar.gz
```
Comment (pinheadmz) [inline download/index.html:129]: we'd need to change the hash as well!

**Comment by pinheadmz** [comment]
hi and thanks again, we're about to release 4.1.0 so I'll close this for now, we will be more diligent going forward.

File: download/index.html (hunk 1 above)
Comment (handshake-enthusiast) [reply download/index.html:129]: sure, that's why WIP with a reference to 667efb0b4a9cff924978849ba532a09c0ca8cd38

**Comment by handshake-enthusiast** [comment]
> hi and thanks again, we're about to release 4.1.0 so I'll close this for now, we will be more diligent going forward.

Sounds great!

**Pull request closed without merging**